### ISSUE TITLE
[ERROR]"Unable to import 'Filters' from 'telegram.ext' in python-telegram-bot library"

### ISSUE THREAD

**Issue by dinesh2sangwan** [python-telegram-bot/python-telegram-bot#4241]
### Issue I am facing

I'm trying to use the `python-telegram-bot` library in my Python script. However, I'm encountering an issue when I try to import the `Filters` class from the `telegram.ext` module. Here's the import statement I'm using:

```python
from telegram.ext import Filters

When I run my script, I get the following error message:

ImportError: cannot import name 'Filters' from 'telegram.ext'



### Traceback to the issue

```python
I’ve confirmed that the python-telegram-bot library is installed in my Python environment, and I can import other classes from the telegram.ext module without any issues. I’ve also checked that the filters.py file (which should contain the Filters class) is present in the telegram/ext/ directory in my site-packages.

I’ve tried uninstalling and reinstalling the python-telegram-bot library, and I’ve also tried creating a new virtual environment, but the issue persists.

I’m using Python 3.10.6 and version 21.1.1 of the python-telegram-bot library. Any help with this issue would be greatly appreciated.

Traceback (most recent call last):
  File "C:\LinuxForDevops\telegram-bot\jodd.py", line 2, in <module>
    from telegram.ext import Updater, CommandHandler, MessageHandler, Filters
ImportError: cannot import name 'Filters' from 'telegram.ext' (C:\LinuxForDevops\telegram-bot\telegram-bot-env\lib\site-packages\telegram\ext\__init__.py)
```

### Operating System

windows 11

### Version of Python, python-telegram-bot & dependencies

```shell
C:\telegram-bot>python -m telegram
python-telegram-bot 21.1.1
Bot API 7.2
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
```

**Comment by Bibo-Joshi** [comment]
Duplicate of #3455

**Comment by dinesh2sangwan** [comment]
> Duplicate of #3544

nahh, not working

**Comment by Bibo-Joshi** [comment]
PTB versions 20+ no longer have on object called telegram.ext.Filters. this was available only in v13.15 and older. This is exactly the problem that was described in the issue that I linked above. Change your code to be compatible with PTB v21.1 or install a version of PTB that is compatible with your code .
